docker/container.sh

```diff
@@ -231,6 +231,24 @@ x11_cleanup() {
     fi
 }
 
+submit_job() {
+
+    echo "[INFO] Arguments passed to job script ${@}"
+    case $CLUSTER_JOB_SCHEDULER in
+        "SLURM")
+            ssh $CLUSTER_LOGIN "cd $CLUSTER_ISAACLAB_DIR && sbatch $CLUSTER_ISAACLAB_DIR/docker/cluster/submit_job_slurm.sh \"$CLUSTER_ISAACLAB_DIR\" \"isaac-lab-$container_profile\" ${@}"
+            ;;
+        "PBS")
+            ssh $CLUSTER_LOGIN "cd $CLUSTER_ISAACLAB_DIR && bash $CLUSTER_ISAACLAB_DIR/docker/cluster/submit_job_pbs.sh \"$CLUSTER_ISAACLAB_DIR\" \"isaac-lab-$container_profile\" ${@}"
```

Comment:
NIT: Instead of duplicating the code, isn't it easier to just resolve the script and put the command to run outside the case? Just to make maintenance easier :)